tests/unit/test_exceptions.py

```diff
@@ -13,17 +13,36 @@
 # limitations under the License.
 
 from unittest import TestCase
-
+import tempfile, os
 from backup_monkey.exception import * 
+from backup_monkey import SplunkLogging
 
 class ExceptionTests(TestCase):
+    log_file = tempfile.mkstemp()[1]
+
+    @classmethod
+    def setUpClass(cls):
+        SplunkLogging.set_path(cls.log_file)
+
     def test_new_exception(self):
         e = BackupMonkeyException()
         self.assertTrue(isinstance(e, BackupMonkeyException))
-        #self.assertIsInstance(e, BackupMonkeyException)
 
     def raise_BackupMonkeyException(self):
-        raise BackupMonkeyException('msg')
-    
+        raise BackupMonkeyException('msg', subject='subject', body='body')
+ 
     def test_raise_BackupMonkeyException(self):
         self.assertRaises(BackupMonkeyException, self.raise_BackupMonkeyException)
+
+    def test_splunk_logging(self):
+        with open(self.log_file) as f:
```

Comment:
Because the file will be clean, the exception needs to be raised before checking the log file.